lunavl/sdk/estimators/face_estimators/mask.py

```diff
@@ -139,31 +131,44 @@ class MaskEstimator(BaseEstimator):
     Warp mask estimator.
     """
 
-    #  pylint: disable=W0235
-    def __init__(self, maskEstimator: IMedicalMaskEstimatorPtr):
-        """
-        Init.
+    #  pylint: disable=W0221
+    @overload  # type: ignore
+    def estimate(self, image: FaceWarp) -> Mask:
+        """Estimate mask from a FaceWarp"""
+        pass
 
-        Args:
-            maskEstimator: core mask estimator
-        """
-        super().__init__(maskEstimator)
+    #  pylint: disable=W0221
+    @overload
+    def estimate(self, image: FaceWarpedImage) -> Mask:
+        """Estimate mask from a FaceWarpedImage"""
+        pass
+
+    #  pylint: disable=W0221
+    @overload
+    def estimate(self, image: Tuple[VLImage, DetectionFloat]) -> Mask:
```

Comment:
wtf/ is image a tuple?

Reply:
I renamed the variable

Reply:
first variant with image and optional detection was better.

Reply:
If used image and optional detection, mypy tell `error: No overload variant of "estimate" of "MaskEstimator" matches argument type "Tuple[VLImage, Any]"`. I can skip this error if you want

Reply:
Can you  show me this case? I mean this api 
```python
    @overload
    def estimate(self, image: VLImage, detection: FaceDetection) -> Mask:
        ...
    @overload
    def estimate(self, image: Union[FaceWarp, FaceWarpedImage]) -> Mask:
        ...
    def  estimate(self, image, detection = None) -> Mask:
         pass
```

Reply:
I fixed it